app/views/layouts/_top_header.html.haml

```diff
@@ -0,0 +1,53 @@
+.service-header
+  .div-nav
+    %nav.navbar.navbar-static-top.navbar-expand-lg.px-5
+      %div
+        = image_tag('coffee.jpg', class: "d-inline-block align-top logo-img")
+        = link_to home_path, id: 'logo-text', class: 'navbar-brand home-page-text' do
+          %div
+            EUREKA
+          %div
+            CAFFEE
+      %button.navbar-toggler{ 'aria-controls' => 'navbar-nav', 'aria-expanded' => 'false', 'aria-label' => 'Toggle navigation', 'data-target' => '#navbar-nav', 'data-toggle' => 'collapse', :type => 'button' }
+        %i.fas.fa-chevron-circle-down
+      #navbar-nav.collapse.navbar-collapse.bg-custom
+        %ul#menu-top.navbar-nav.ml-auto          
+          %li.nav-item.active
```

Comment:
this looks a bit messy, can we do it nicer, I don't know, use partials for something for different kind of navigation depending on the user?

Reply:
started making use of partials, making code more readable & clear. Was not sure whether or not code logic also needed changing, so pushed the changes regarding partials for now. Bump me up if further changes are needed regarding this issue. ^_^